src/main/java/com/benko/exercise/sort/SortingApp.java

```diff
@@ -0,0 +1,11 @@
+package com.benko.exercise.sort;
+
+public class SortingApp {
+
+    public static void main(String[] args) {
+
```

Comment:
remove this empty line

Reply:
Done

Reply:
I meant the line in the main() method :)
ie. between method name and first method statement
the line between the class and method was fine and in fact people often use it

Reply:
heh I was a bit surprised because I think that I belong to this people 👍 
will do next time